core/dbt/utils.py

```diff
@@ -600,14 +601,15 @@ def __contains__(self, name) -> bool:
 
 def _connection_exception_retry(fn, max_attempts: int, attempt: int = 0):
     """Attempts to run a function that makes an external call, if the call fails
-    on a connection error or timeout, it will be tried up to 5 more times.
+    on a connection error, timeout or other exception, it will be tried up to 5 more times.
```

Comment:
are "other exceptions" all related to decoding/reading? should this comment be more specific? It might be able to provide historical context by including a ticket number.